### PR TITLE
conda-forge package for RNMC_native

### PR DESCRIPTION
**Pull request by danielbarter**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by danielbarter** [comment]
Hey @conda-forge/help-c-cpp, this package is ready for review.

File: recipes/rnmc-native/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "./build.sh RNMC"
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - gsl
```
Comment (dopplershift) [inline recipes/rnmc-native/meta.yaml:17]: You shouldn't need `gsl` as both build & host dependency. It should be enough to have it under `host`.

Comment (danielbarter) [reply recipes/rnmc-native/meta.yaml]: yep, seems to work. Thanks @dopplershift

**Comment by dopplershift** [comment]
So it looks like this is directly downloading a tarball from a particular git hash? Is there any way to get a tagged release on that repository that matches the 1.0 version that's listed here?

**Comment by danielbarter** [comment]
Yeah, for sure. Will do it now

**Comment by danielbarter** [comment]
OK, that should be better now. Sorry for the delay on that @dopplershift

**Review by dopplershift — APPROVED** [review]
No worries! Thanks for the contribution.

**Pull request merged**